### ISSUE TITLE
[dockerhub] backend enrichment error (python exception)

### ISSUE THREAD

**Issue by lukaszgryglicki** [chaoss/grimoirelab-elk#767]
Python exception:

`p2o.py --enrich --index raw --index-enrich rich -e [redacted_es_7_url] -g --db-host mariadb.mariadb --db-sortinghat sortinghat --db-user sortinghat --db-password [redacted] dockerhub hyperledger sawtooth-next-directory`:

```
2020-01-16 13:48:53,765 Error enriching raw from dockerhub (https://hub.docker.com/hyperledger/sawtooth-next-directory): unorderable types: NoneType() <= NoneType()
Traceback (most recent call last):
  File "/repos/grimoirelab-elk/grimoire_elk/elk.py", line 666, in enrich_backend
    enrich_count = enrich_items(ocean_backend, enrich_backend)
  File "/repos/grimoirelab-elk/grimoire_elk/elk.py", line 452, in enrich_items
    total = enrich_backend.enrich_items(ocean_backend)
  File "/repos/grimoirelab-elk/grimoire_elk/enriched/dockerhub.py", line 166, in enrich_items
    if image_date <= rich_item['last_updated']:
TypeError: unorderable types: NoneType() <= NoneType()
2020-01-16 13:48:53,766 [dockerhub] Done enrichment for https://hub.docker.com/hyperledger/sawtooth-next-directory
2020-01-16 13:48:53,767 Enrich backend completed
2020-01-16 13:48:53,767 Finished in 0.03 min
```

**Comment by valeriocos** [comment]
Fixed with #774

**Comment by lukaszgryglicki** [comment]
Thanks!
